src/index.js

```diff
@@ -1,6 +1,22 @@
 let _ = require('lodash/fp')
 let { getESSchemas } = require('./schema')
 let redisCache = require('./utils/redisCache')
+let debug = require('debug')('contexture:elasticsearch')
+
+let revolvingCounter = max => {
+  let counter = 0
+  return {
+    inc() {
+      if (counter === max) {
+        counter = 1
+      } else {
+        counter++
+      }
+      return counter
+    },
+  }
+}
+let counter = revolvingCounter(500)
```

Comment:
I'm not sure if we need the above. The thought behind it was that large numbers are not super helpful when debugging.